### DISCUSSION TITLE
Extension Management and JSON in Intune

### DISCUSSION THREAD

**Discussion by marius127** [mozilla/policy-templates#982] [Q&A]
Hi,

The description under the "extension management" option says I can use JSON to configure/describe to policy. However, when using Intune and the new "admx import" option you are not allowed to enter more than 1023 characters in the text field. 

Does anyone know how to solve this? 

Works fine when using the Custom OMA-URI Settings option, but I prefer the new "admx import" alternative. 

![Screenshot](https://user-images.githubusercontent.com/90035286/206553103-9d985977-66bc-4b51-896e-c92b3b2198e9.png)

**Comment by mkaply** [comment]
I've reported this bug to Microsoft. They aren't handling multiText correctly.

**Comment by mkaply** [comment]
I added a "JSON one line" to work around this.
